components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -33,3 +36,9 @@ fun Context.isOSOnLowMemory(): Boolean {
         activityManager.getMemoryInfo(memoryInfo)
     }.lowMemory
 }
+
+/**
+ * AccessibilityManager.isAccessibilityEnabled can be enabled for more things than just VoiceView. If we want
+ * VoiceView, we only need to check this one field; via comments on https://stackoverflow.com/a/12362545/582004.
+ */
+fun Context.isScreenReaderEnabled() = getAccessibilityManager()?.isTouchExplorationEnabled
```

Comment:
This method returns an nullable Boolean. This makes it awkward to use. I guess if there's no accessibility manager available then we can just return false?

Reply:
That would make more sense